src/harness/harnessIO.ts

```diff
@@ -847,12 +847,28 @@ namespace Harness {
                     result.maps.forEach(sourceMap => {
                         if (sourceMapCode) sourceMapCode += "\r\n";
                         sourceMapCode += fileOutput(sourceMap, harnessSettings);
+                        if (!options.inlineSourceMap) {

```

Comment:
If this is anyways going to be part of json file, canyou instead move emitting .map files into jsEmit baselines.. (we reduce number of files plus its easier to look at as well and get read of sourceMap.baseline.txt?)

Reply:
Cool.  These changes got merged in #37111 - but I'm open to changing how it works!

Just to make sure I get the idea: This would be editing`doJsEmitBaseline` to also include the sourcemap and link to the sourcemap file at the bottom of the emitted JS files as another section? (then I delete all the sourcemap files from test suite)